### PR TITLE
Add list short feature

### LINKED ISSUE #73: As a Property Manager, I can limit the number of list results
... so that my CLI window is not overwhelmed by list results

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -165,29 +165,36 @@ public void displayOneClient(Client client, int i) {
     }
 
     public void displayOneClientName(Client client, int i) {
-        System.out.println(i + ".");
+        System.out.print(i + FULL_STOP);
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Ui.java:168]: Good use of string constants!

File: src/main/java/seedu/duke/command/CommandListEverything.java [hunk 2]
```diff
@@ -13,13 +13,11 @@ public void execute(Ui ui, Storage storage, PropertyList propertyList, ClientLis
         for (int i = 0; i < propertyList.getCurrentListSize(); i++) {
             ui.displayOneProperty(propertyList.getPropertyList().get(i), i + 1);
         }
-        System.out.println("There are " + propertyList.getCurrentListSize()
-                + " properties in this list");
+        ui.displayNoOfProperties(propertyList.getCurrentListSize());
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/command/CommandListEverything.java:16]: Good job converting printing code into reusable methods

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/duke/command/CommandListClientsWithTags.java [hunk 3]
```diff
@@ -39,32 +42,35 @@ private void displayClientContacts(ClientList clientList, Ui ui) {
...
             ui.displayOneClientContact(clientList.getClientList().get(i), i + 1);
         }
-        System.out.println("There are " + clientList.getCurrentListSize()
-                + " clients in this list");
+        ui.displayNoOfClients(clientList.getCurrentListSize());
+    }
+
+    public void displayClientShort(ClientList clientList, Ui ui) {
+        for (int i = 0; i < clientList.getCurrentListSize(); i++) {
+            ui.displayOneClientShort(clientList.getClientList().get(i), i + 1);
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/command/CommandListClientsWithTags.java:50]: Do avoid using magic literals

**Review by FeliciaBeatrice — APPROVED** [review]
everything looks good! LGTM

**Pull request merged**